CDTS Blazor/Startup.cs

```diff
@@ -36,8 +36,10 @@ public static void ConfigureServices(IServiceCollection services)
         {
             services.AddRazorPages();
             services.AddServerSideBlazor();
-            services.AddSingleton<IAzureBlobConnectionFactory, AzureBlobConnectionFactory>();
-            services.AddTransient<IAzureBlobService, AzureBlobService>();
+
+            services.AddSingleton(new AzureKeyVaultService("https://kv-seafarer-dev.vault.azure.net/"));
```

Comment:
This will value will need to be properly configured by environment.

Reply:
We can make a PBI for it

Reply:
Sounds great!